### ISSUE TITLE
Add a changelog file

### ISSUE THREAD

**Issue by jpggithub** [orgzly-revived/orgzly-android-revived#173]
I use Orgzly from f-droid and I found the changelog files useful to know the enhancements of softwares I download on my phone.

Perhaps you can consider add one for Orgzly.

**Comment by amberin** [comment]
This should appear in the next upcoming release, 1.8.18. At least I've added files to the /metadata directory - let's see if it works.

**Comment by stefan2904** [comment]
btw, we forgot to update [`dialog_whats_new.xml`](https://github.com/orgzly-revived/orgzly-android-revived/commits/master/app/src/main/res/layout/dialog_whats_new.xml)in the 1.8.17 release.

Could this be automated since the info is already in the GitHub changelog? 
(and/or synced with the changelog in `./metadata`?)

**Comment by amberin** [comment]
Yes, I'd love to hear ideas on how to automate it - I keep forgetting to do the manual updates. (I've added the v1.8.17 changes in the 1.8.18 release branch.)

**Comment by jpggithub** [comment]
Up! As we are at the version 1.12.1 on f-droid 😉

**Comment by amberin** [comment]
Yes, I created those metadata files but forgot to include them in the 1.13.1 commit. :( The info should appear with the next F-Droid release.
